paasta_tools/setup_kubernetes_job.py

```diff
@@ -135,8 +136,10 @@ def setup_kube_deployments(
         for service_instance in service_instances_with_valid_names
     ]
 
+    apps_updated = []
     for _, app in applications:
         if app:
+            apps_updated.append(str(app))
             if (
                 app.kube_deployment.service,
                 app.kube_deployment.instance,
```

Comment:
Can we make all the "debug" messages (except for the "app is up to date" message) `info`-level instead of `debug`-level?  That will be really helpful for understanding why changes are getting made.